crates/devices/virtio-vsock/Cargo.toml

```diff
@@ -10,9 +10,9 @@ license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2018"
 
 [dependencies]
-virtio-queue = "0.2.0"
-vm-memory = "0.7.0"
+virtio-queue = { path = "../../virtio-queue" }
```

Comment:
Should this be
```
virtio-queue = { version = "0.3", path = "../../virtio-queue" }
```

Reply:
It should, but we won't be allowed to publish it anyway, so I was thinking to update it at that point.

Reply:
@jiangliu I just make it consistent with the other crates.